applications/FluidDynamicsApplication/custom_strategies/schemes/residualbased_simple_steady_scheme.h

```diff
@@ -213,6 +213,17 @@ class ResidualBasedSimpleSteadyScheme : public Scheme<TSparseSpace, TDenseSpace>
     KRATOS_CATCH("");
   }
 
+    void InitializeNonLinIteration(
+        ModelPart &rModelPart,
+        TSystemMatrixType &A,
+        TSystemVectorType &Dx,
+        TSystemVectorType &b) override
+    {
+
+        BaseType::InitializeNonLinIteration(rModelPart, A, Dx, b);
+
+    }
+
```

Comment:
This is doing nothing. Please remove it and leave current base class implementation.